sos/component.py

```diff
@@ -256,7 +256,7 @@ def setup_archive(self, name=''):
                                           enc_opts, self.sysroot,
                                           self.manifest)
 
-        self.archive.set_debug(True if self.opts.debug else False)
+        self.archive.set_debug(True if self.opts.verbosity > 2 else False)
```

Comment:
A monkey questions: why not just `self.archive.set_debug(self.opts.verbosity > 2)` ?